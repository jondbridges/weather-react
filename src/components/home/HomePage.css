.HomePage-content {
  background-size: cover;
  background-image: url('/images/pattern.svg');
  display: flex;
  align-items: center;
  height: 92%;
  width: 100%;
}

.HomePage-search-label {
  color: white;
  font-weight: 100;
  font-size: 35px;
}
