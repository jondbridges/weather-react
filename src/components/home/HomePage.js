import React from 'react';

class HomePage extends React.Component {
  render() {
    return (
      <div>
        Home Page, Main Content
      </div>
    );
  }
}

export default HomePage;
