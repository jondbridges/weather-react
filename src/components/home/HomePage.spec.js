import React from 'react';
import HomePage from './HomePage';
import renderer from 'react-test-renderer';

describe('HomePage', () => {

  it('creates the home page content', () => {
    const component = renderer.create(
      <HomePage />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
