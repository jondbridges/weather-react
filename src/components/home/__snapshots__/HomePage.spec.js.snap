exports[`HomePage creates the home page content 1`] = `
<div
  className="HomePage-content">
  <div
    className="col-md-4 col-md-offset-4 text-center">
    <div
      className="HomePage-search-label">
      Enter a City and State
    </div>
    <div>
      <div
        className="form-group">
        <input
          className="form-control"
          onChange={[Function anonymous]}
          placeholder="St Louis, MO"
          type="text" />
      </div>
      <button
        className="btn btn-success Search-btn-pad-left"
        onClick={[Function anonymous]}
        type="submit">
        Get Weather
      </button>
    </div>
  </div>
</div>
`;
